openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -33,16 +35,64 @@
 ]
 
 
-def get_featured_course():
+def get_course_card_data(course, org_prefetched=False):
+    """
+    Get course data required for home page course card
+
+    :returns (dict): dict of course card data
+    """
+    pakx_short_logo = '/static/pakx/images/mooc/pX.png'
+    course_custom_setting = getattr(course, 'custom_settings', None)
+
+    if not org_prefetched:
+        course_org = Organization.objects.filter(name__iexact=course.org).first()
```

Comment:
shouldn't we use org's `short_name` here, usually courses org are like pakx, btc, ttc